WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/IoT/AthensAppx.cs

```diff
@@ -0,0 +1,152 @@
+﻿//----------------------------------------------------------------------------------------------
+// <copyright file="AthensAppx.cs" company="Microsoft Corporation">
+//     Licensed under the MIT License. See LICENSE.TXT in the project root license information.
+// </copyright>
+//----------------------------------------------------------------------------------------------
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+
+namespace Microsoft.Tools.WindowsDevicePortal
+{
+    /// <content>
+    /// Wrappers for Application Management.
+    /// </content>
+    public partial class DevicePortal
+    {
+        /// <summary>
+        /// IoT device application list API.
+        /// </summary>
+        public static readonly string AppsListApi = "api/IoT/appx/default";
+
+        /// <summary>
+        /// IoT device headless application list API.
+        /// </summary>
+        public static readonly string HeadlessAppsListApi = "api/iot/appx/listHeadlessApps";
+
+        /// <summary>
+        /// IoT device headless startup application API.
+        /// </summary>
+        public static readonly string HeadlessStartupAppApi = "api/iot/appx/startupHeadlessApp";
+
+        /// <summary>
+        /// IoT device package activation API.
+        /// </summary>
+        public static readonly string ActivatePackageApi = "api/iot/appx/app";
+
+        /// <summary>
+        /// Gets List of apps.
+        /// </summary>
+        /// <returns>String containing the list of applications.</returns>
```

Comment:
String? Might make more sense to either return the array itself or say "Object containing a list of applications".